vagrant/config.sh

```diff
@@ -42,11 +42,10 @@ sed -i 's/127.0.0.1\/32/0.0.0.0\/0/g' /media/data/pg_data/data/pg_hba.conf
  
 echo "listen_addresses = '*'" >> /media/data/pg_data/data/postgresql.conf
 sed -i 's/shared_buffers = 128MB/shared_buffers = 4GB/g' /media/data/pg_data/data/postgresql.conf
-echo "shared_preload_libraries = 'pg_session_stats'" >> /media/data/pg_data/data/postgresql.conf
-echo "pg_session_stats.path = '/media/data/pg_data/pgss.sqlite3'" >> /media/data/pg_data/data/postgresql.conf
-
-systemctl stop postgresql
+#echo "shared_preload_libraries = 'pg_session_stats'" >> /media/data/pg_data/data/postgresql.conf
+#echo "pg_session_stats.path = '/media/data/pg_data/pgss.sqlite3'" >> /media/data/pg_data/data/postgresql.conf
 
+#systemctl stop postgresql
```

Comment:
Should be `systemctl restart postgresql`

Reply:
Right. My bad. Done!